install-gtex.sh

```diff
@@ -167,6 +168,12 @@ UNINSTALL_SCRIPT="${TEXMFROOT}/${UNINSTALL_SCRIPT_DIR}/${UNINSTALL_SCRIPT_FILE}"
 
 function die {
     echo 'Failed.'
+    if $install_start
+    then
+        echo "Cleaning up partial install"
```

Comment:
This should probably check that the uninstall script exists before trying to run it.

Reply:
Will do.  I forgot there was a way to prevent the uninstall script from being created during the installation process.